File: Develop/src/Bank.java
import java.util.ArrayList;
import java.util.Random;
public class Bank {
    int k=0;
    public ArrayList<Employee> getClientes() {
        return Clientes;
    }

    public ArrayList<Employee>Clientes=new ArrayList<>();

    public void clientes(Employee cliente){
        Clientes.add(cliente);
    }


    public void imprimirClientes(){
        for (int i=0;i<Clientes.size();i++) {
            System.out.printf("%s %s No %d Type:%c Balance:%f\n" ,Clientes.get(i).getName(),Clientes.get(i).getLastName(),Clientes.get(i).getAccount().getAccountNumber(),Clientes.get(i).getAccount().getType(),Clientes.get(i).getAccount().getAmount());

        }
    }
    public void info(int espacio){
        espacio=k;
        if(espacio>=0){
            System.out.printf("%s %s No %d Type:%c Balance:%f\n" ,Clientes.get(k).getName(),Clientes.get(k).getLastName(),Clientes.get(k).getAccount().getAccountNumber(),Clientes.get(k).getAccount().getType(),Clientes.get(k).getAccount().getAmount());}


    }
    public void buscarUser(String nombre, String lastName){
        for (int i=0;i<Clientes.size();i++) {
            if (Clientes.get(i).getName().equals(nombre)){
                if (Clientes.get(i).getLastName().equals(lastName)){
                    System.out.printf("%s %s No %d Type:%c Balance:%f\n" ,Clientes.get(i).getName(),Clientes.get(i).getLastName(),Clientes.get(i).getAccount().getAccountNumber(),Clientes.get(i).getAccount().getType(),Clientes.get(i).getAccount().getAmount());


                }
            }
        }
    }
    public long validacionNum(long acc){
        for (int i=0;i<Clientes.size();i++) {
            if (Clientes.get(i).getAccount().getAccountNumber()==acc){
                Random num=new Random();
                acc=num.nextLong(10)+1;
                i=0;
            }
        }return  acc;


    }

    public Employee logIn(String name,String lastname,long acc){
        Employee personanew=null;
        String n=name,ln=lastname;long nc=acc;
        for (int k = 0; k <Clientes.size(); k++) {
            if (Clientes.get(k).getName().equals(n)) {
                if (Clientes.get(k).getLastName().equals(ln)) {
                    if(Clientes.get(k).getAccount().getAccountNumber()==nc) {
                        personanew=Clientes.get(k);
                        this.k=k;
                        info(this.k);
                        System.out.printf("%s %s No %d Type:%c Balance:%f\n" ,Clientes.get(k).getName(),Clientes.get(k).getLastName(),Clientes.get(k).getAccount().getAccountNumber(),Clientes.get(k).getAccount().getType(),Clientes.get(k).getAccount().getAmount());}
                }
            }
        }

        return personanew;
    }
}

File: Bank.py
import random

class Bank:
    def __init__(self):
        self.k = 0
        self.clientes = []

    def get_clientes(self):
        return self.clientes

    def clientes(self, cliente):
        self.clientes.append(cliente)

    def imprimir_clientes(self):
        for cliente in self.clientes:
            print(f"{cliente.get_name()} {cliente.get_last_name()} No {cliente.get_account().get_account_number()} Type:{cliente.get_account().get_type()} Balance:{cliente.get_account().get_amount()}")

    def info(self, espacio):
        espacio = self.k
        if espacio >= 0:
            cliente = self.clientes[self.k]
            print(f"{cliente.get_name()} {cliente.get_last_name()} No {cliente.get_account().get_account_number()} Type:{cliente.get_account().get_type()} Balance:{cliente.get_account().get_amount()}")

    def buscar_user(self, nombre, last_name):
        for cliente in self.clientes:
            if cliente.get_name() == nombre and cliente.get_last_name() == last_name:
                print(f"{cliente.get_name()} {cliente.get_last_name()} No {cliente.get_account().get_account_number()} Type:{cliente.get_account().get_type()} Balance:{cliente.get_account().get_amount()}")

    def validacion_num(self, acc):
        for cliente in self.clientes:
            if cliente.get_account().get_account_number() == acc:
                acc = random.randint(1, 10)
                return acc
        return acc

    def log_in(self, name, last_name, acc):
        personanew = None
        n, ln, nc = name, last_name, acc
        for k, cliente in enumerate(self.clientes):
            if cliente.get_name() == n and cliente.get_last_name() == ln and cliente.get_account().get_account_number() == nc:
                personanew = cliente
                self.k = k
                self.info(self.k)
                print(f"{cliente.get_name()} {cliente.get_last_name()} No {cliente.get_account().get_account_number()} Type:{cliente.get_account().get_type()} Balance:{cliente.get_account().get_amount()}")
        return personanew
